frontend/src/components/elements/Balloons/Balloons.scss

```diff
@@ -14,51 +14,37 @@
  * limitations under the License.
  */
 
-.balloons > .balloon {
-  position: fixed;
-  bottom: 0;
-  margin-left: -68px; /* Center 136px-wide image */
-  z-index: 1000000;
-
-  /* If modifying the duration, you should also modify Balloons.js */
-  animation-duration: 0.3s, 1s, 1s;
-  animation-name: wobble, fade, move-up;
-  animation-timing-function: ease-in-out, ease-in, ease-in;
-  animation-direction: alternate, normal, normal;
-  animation-iteration-count: infinite, 1, 1;
-  opacity: 0;
-}
-
-@keyframes move-up {
-  from {
-    bottom: 0;
-  }
-
-  to {
-    bottom: 110vh;
-  }
-}
-
-@keyframes fade {
-  0% {
-    opacity: 0;
-  }
-
-  80% {
-    opacity: 1;
-  }
-
-  100% {
+@import "src/assets/css/variables";
+
+$imageHeight: 300px;
+$imageWidth: 121px;
+
+.balloons {
+  .balloon {
+    position: fixed;
+    top: -$imageHeight;
+    margin-left: -$imageWidth / 2;
+    z-index: $z-index-balloons;
+
+    height: $imageHeight;
+    width: $imageWidth;
+
+    /* If modifying the duration, you should also modify Balloons.js */
```

Comment:
I was confused, but it's `Balloons.tsx` (Balloons.js sounds like a library lol).

In any case, we should maybe make it a `SCSS_VAR` to avoid two locations?

Reply:
Ooops, this comment is actually old. I removed the code in JS that depended on this, as it was causing some jankiness under certain conditions.

Reply:
Removed the comment now.